docs/user/power.md

```diff
@@ -0,0 +1,64 @@
+### Statistical Power Plots
+
+`plot_pwr0_statistic` simply plots statistical calculation of `pwr0`, lag 0, in the loaded in RAWACF file. 
```

Comment:
```suggestion
`plot_pwr0_statistic` plots various statistical calculations of `pwr0`, lag 0, in the selected RAWACF file. 
```

Reply:
Maybe ask Cooper to add a little spiel about why this function is useful and in what cases?

Reply:
@Cooper10  if you're up for it that is! you're the pro